Multisig/Data/Services/SafeTransactionService.swift

```diff
@@ -55,8 +55,16 @@ class SafeTransactionService {
         return try httpClient.execute(request: SafeBalancesRequest(address: address))
     }
 
-    func transactions(address: Address, offset: Int = 0, limit: Int = 100) throws -> TransactionsRequest.Response {
-        return try httpClient.execute(request: TransactionsRequest(address: address, limit: limit, offset: offset))
+    func transactions(address: Address?, offset: Int = 0, limit: Int = 20) throws -> TransactionsRequest.Response {
+        return try httpClient.execute(request: TransactionsRequest(address: address!, limit: limit, offset: offset))
+    }
+
+    func loadTransactionsPage(url: String) throws -> TransactionsRequest.Response {
+        let request = PagedRequest<Transaction>(url)
+
+        assert(request != nil)
+
+        return try httpClient.execute(request: request!)
```

Comment:
I'd prefer not to crash the app in production, if the url returned by backend is invalid. I think it is an expected case and must be handled appropriately

Reply:
I would use `let request = PagedRequest<Transaction>(url)!`
If it crashes here, I would treat it as a developer error